permissions/plugin-github-checks.yml

```
@@ -0,0 +1,9 @@
+---
+name: "github-checks"
+github: "jenkinsci/github-checks-plugin"
+paths:
+- "org/jenkins-ci/plugins/github-checks-plugin"
```

https://github.com/jenkinsci/github-checks-plugin/blob/master/pom.xml#L13

```suggestion
- "io/jenkins/plugins/github-checks"
```